src/Command/SelfUpdateCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace SensioLabs\Deptrac\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+class SelfUpdateCommand extends Command
+{
+    public function configure()
+    {
+        $this
+            ->setName('self-update')
+            ->setAliases(array('selfupdate'))
```

Comment:
At least, having this alias would be inline what Composer and the Symfony Installer offer (thus people might be used to use `selfupdate` instead of `self-update`).